src/components/InventoryTable/InventoryTable.js

```diff
@@ -201,7 +201,7 @@ const InventoryTable = forwardRef(({ // eslint-disable-line react/display-name
                     paginationProps={paginationProps}
                     loaded={loaded}
                     showTagModal={showTagModal}
-                    activeFiltersConfig={{ deleteTitle: 'Reset filters', ...props.activeFiltersConfig, showDeleteButton: true }}
+                    activeFiltersConfig={{ deleteTitle: 'Reset filters', ...props.activeFiltersConfig }}
```

Comment:
Let's add a test for when the InventoryTable has a `activeFiltersConfig` prop set that it pass it along to the EntityTableToolbar.

Reply:
Just added